modules/core/src/main/java/org/apache/synapse/deployers/APIDeployer.java

```diff
@@ -60,8 +61,12 @@ public String deploySynapseArtifact(OMElement artifactConfig, String fileName, P
                 if (log.isDebugEnabled()) {
                     log.debug("API deployment from file : " + fileName + " : Completed");
                 }
+                String startTime  = String.valueOf(System.currentTimeMillis());
                 log.info("API named '" + api.getName() +
                         "' has been deployed from file : " + fileName);
+                //Executing the extended Synapse Handler during API deployment
+                executeExtendedSynapseHandlerOnArtifactDeployment(api.getName(), SynapseConstants.FAIL_SAFE_MODE_API,
```

Comment:
does the time for log printing matters. shall we make these 64 and 68 together or inline.